src/components/Messages/extensions/Channel.js

```diff
@@ -54,7 +56,9 @@ const Channel = ({ user = null, db = null }) => {
         displayName,
         photoURL,
       });
+      // onclick="document.getElementById('myInput').value = ''"
```

Comment:
try this instead:
    if (db) {
      db.collection("messages").add({
        text: newMessage,
        createdAt: firebase.firestore.FieldValue.serverTimestamp(),
        uid,
        displayName,
        photoURL,
      }).then(() => {
        setNewMessage(() => {
          return ''
        })
      })
    }